cmd/gpud/command/run.go

```diff
@@ -30,15 +30,20 @@ func cmdRun(cliContext *cli.Context) error {
 
 	var zapLvl zap.AtomicLevel = zap.NewAtomicLevel() // info level by default
 	if logLevel != "" && logLevel != "info" {
-		lCfg := log.DefaultLoggerConfig()
 		var err error
 		zapLvl, err = zap.ParseAtomicLevel(logLevel)
 		if err != nil {
 			return err
 		}
+	}
+	if logFile != "" {
+		log.Logger = log.CreateLoggerWithLumberjack(logFile, 128, zapLvl.Level())
+	} else {
+		lCfg := log.DefaultLoggerConfig()
```

Comment:
The gpud systemd service is not changed in this PR, so I'm wondering how we plan to upgrade the systemd service to remove the stdout/err appen parameters?

https://github.com/leptonai/gpud/blob/2175d7d3671ccde724a6145ddbb87aa3f36e2cbd/pkg/gpud-manager/systemd/gpud.service#L14-L15

Reply:
For now, this is optional (only enabled when `run --log-file` is not empty /cc @cardyok 

We can drop this once we migrate